npm/npm_import.bzl

```diff
@@ -225,13 +237,35 @@ def npm_import(
             At best omitting this field will make your build non-hermetic.
 
             It is optional to make development easier but should be set before shipping.
+
         url: Optional url for this package. If unset, a default npm registry url is generated from
             the package name and version.
+
         patch_args: Arguments to pass to the patch tool.
             `-p1` will usually be needed for patches generated by git.
+
         patches: Patch files to apply onto the downloaded npm package.
+
         custom_postinstall: Custom string postinstall script to run on the installed npm package. Runs after any
             existing lifecycle hooks if `run_lifecycle_hooks` is True.
+
+        bins: Dictionary of `node_modules/.bin` binary files to create mapped to their node entry points.
+
+            This is typically derived from the "bin" attribute in the package.json
+            file of the npm package being linked.
+
+            For example:
+
+            ```
+            bins: {
+                "foo": "./foo.js",
```

Comment:
Shouldn't the value be in an array? And have the `=`?

Is there a better package name then "foo" that clearly indicates that that is the package? Even just `pkg-foo`?

Reply:
Oh I see, this is the `npm_import`, and it does align with the `npm_translate_lock` 👍

Reply:
Yeah, unfortunately there is no `string_list_dict_dict` so the equals is a work-around in npm_translate_lock.

Reply:
Tho I might do the translate from string_list_dict_dict in wrapper macro instead so the API is consistent

Reply:
✅